tools/pushDocker

```diff
@@ -6,3 +6,6 @@ echo "BRANCH=$BRANCH."
 docker push kodekonveyor/toolchain:$BUILD_NUMBER 
 docker push kodekonveyor/toolchain:$(echo $BRANCH|sed 'sA/A.Ag')
 
+docker push kodekonveyor/toolchain:developer-$BUILD_NUMBER
```

Comment:
tools/testenv (the script putting the developer to the build environment) should be modified to boot the developer image based on the build image version found in shippable.yml.

Reply:
you mean in projects like cdd, for example? But only when shippable changes to a version of toolchain that has a developer tag, right?

Reply:
yes. What we do now is that we have a testenv script in the toolchain. But because it is not seen before entering the development environment, it is copied to each of the projects. So in the project you can just enter the testenv, and upgrade this script by copying it over form where it is within /usr/local/toolchain.
This is not a good solution, just one I could came up fast.

Reply:
fine